### DISCUSSION TITLE
efficiency/mac/surge

### DISCUSSION THREAD

**Discussion by giscus** [yqchilde/yqchilde.github.io#7] [Q&A]
# efficiency/mac/surge

对我而言，Surge就是MacOS上最好用的网络科学工具

https://yqqy.top/efficiency/mac/surge

**Comment by max1874** [comment]
博客好看，请问大佬是怎么搭建的

**Reply by yqchilde** [reply]
用的是vitepress😄

**Comment by daniellauyu** [comment]
哥，我只想说一句我爱你，看了妈的那么多教程，就你的让我成功了
